simcon_project/users/views/view_response.py

```diff
@@ -0,0 +1,90 @@
+from django.shortcuts import render, get_object_or_404
+# from django.contrib.auth.decorators import login_required, permission_required
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+
+from users.forms import UpdateFeedback
+from conversation_templates.models import TemplateResponse, TemplateNodeResponse
+
+
+def ViewResponse(request, pk=None):
+    response = get_object_or_404(TemplateResponse, pk=pk)
+
+    nodes = []
+    num_nodes = TemplateNodeResponse.objects.filter(parent_template_response=response).count()
+    for i in range(1, num_nodes+1):
```

Comment:
why plus 1? If theres 10 responses wouldn't u just want 1-10 ?? or 0-9??

Reply:
Because ranges don't use the last number so range(1, 11) goes up to 10